tests/acceptance/acceptance_test.go

```diff
@@ -40,7 +40,7 @@ var _ = Describe("Acceptance", func() {
 
 		BeforeEach(func() {
 			profileURL = "https://github.com/weaveworks/nginx-profile"
-			branch = "main"
+			branch = "move-helm-to-nested"
```

Comment:
another TODO

Reply:
yeah its all over to get the tests passing